### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by bmielnicki**
I've added basic integration tests and commented out unused code with error (undefined variable is used).

### CODE REVIEW

**Comment by micahcarroll** [comment]
Also feel free to add me as a reviewer to any new PRs you make!

File: .github/workflows/pythontests.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        python-version: '3.7' 
+        architecture: 'x64'
+    - name: Install dependencies
+      run: python -m pip install --upgrade pip
+    - name: Setup
+      run: python setup.py develop
+    - name: Run tests
+      run: |
+        cd overcooked_ai_py
+        python run_tests.py
```
[the remark this answers is not in the document]
Comment (bmielnicki) [reply .github/workflows/pythontests.yml:23]: There is possibility to run workflow on schedule e.g. every day or when anything in selected folder e.g. overcooked_ai_py/planning was changed or both. On the other hand running full test suite or both tests every time does not exceed github limits anyway.

File: overcooked_ai_py/agents/benchmarking.py [hunk 2]
```diff
@@ -373,11 +373,12 @@ def update(t = 1.0):
 
         t = widgets.IntSlider(min=0, max=len(states) - 1, step=1, value=0)
         out = interactive_output(update, {'t': t})
-        display(out, t)
+        # TODO: add function display; flake8 sees here undefined name "display"
+        # display(out, t)
```
[the remark this answers is not in the document]
Comment (bmielnicki) [reply overcooked_ai_py/agents/benchmarking.py:377]: I think import is the most elegant solution - it gives explicit information where display comes from.

Where does display function come from? Would it be problematic to add an import to the top of the file (with adding dependency to the setup file if needed)?

[the remark this answers is not in the document]
Comment (micahcarroll) [reply overcooked_ai_py/agents/benchmarking.py:377]: I think this should be the import: `from IPython.display import display`

Adding at the top of the file seems fine. The reason it wasn't already there is that when using a Jupyter notebook, this function is imported by default

**Pull request merged**